core/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,14 +1,3 @@
 databaseChangeLog:
-
-  - changeSet:
-      id: _fakeId
-      author: _fakeAuthor
-      changes:
-        - createTable:
-            tableName: _fakeTable
-            remarks: _fake
-            columns:
-              - column:
-                  name: _fakeId
-                  type: integer
-                  autoIncrement: true
\ No newline at end of file
+  - include:
+      file: classpath:db/changelog/db.ctb-4.changelog.postgresql.sql
```

Comment:
Migrations should be splitable
Store all the migrations in one file - bad practice